common/src/main/scala/explore/components/primereact/FormEnumDropdownView.scala

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.components.primereact
+
+import cats.syntax.all.*
+import crystal.react.View
+import eu.timepit.refined.types.string.NonEmptyString
+import japgolly.scalajs.react.vdom.html_<^.*
+import japgolly.scalajs.react.*
+import lucuma.core.util.Display
+import lucuma.core.util.Enumerated
+import react.common.*
+import react.primereact.PrimeStyles
+import reactST.primereact.selectitemMod.SelectItem
+
+import scalajs.js
+import scalajs.js.JSConverters.*
+import lucuma.core.enums.Half.A
+
+final case class FormEnumDropdownView[A](
```

Comment:
For this and the option drop down view for enums, the only reason to have a `Form` version and a `non-Form` version is that the `non-Form` version can return the builder, while the `Form` version can't because it also has a label. If we do a full facade or don't expose the builder, there is probably no real reason to have the `non-Form` variant.